tests/sentry/rules/processing/test_delayed_processing.py

```diff
@@ -64,9 +122,121 @@ def test_bulk_fetch_events(self):
+        fake_data_groups = DataAndGroups(
+            data=self.create_event_frequency_condition(id="fake_id"),
+            group_ids={1},
+        )
+        assert get_condition_group_results({first_query: fake_data_groups}, self.project) is None
+        mock_logger.warning.assert_called_once()
+
+    @patch("sentry.rules.processing.delayed_processing.logger")
+    def test_fast_condition_cls(self, mock_logger):
+        first_query = UniqueConditionQuery(
+            cls_id="sentry.rules.conditions.every_event.EveryEventCondition",
+            interval="",
+            environment_id=1,
+        )
+        fake_data_groups = DataAndGroups(
+            data=self.create_event_frequency_condition(id="fake_id"),
+            group_ids={1},
+        )
+        assert get_condition_group_results({first_query: fake_data_groups}, self.project) is None
+        mock_logger.warning.assert_called_once()
+
+    def test_group_does_not_belong_to_project(self):
+        condition_data = self.create_event_frequency_condition(interval=self.interval)
+        condition_groups, group_id, unique_queries = self.create_condition_groups([condition_data])
+
+        results = get_condition_group_results(condition_groups, self.create_project())
+        assert results == {
+            unique_queries[0]: {group_id: 2},
+        }
+
+    def test_count_comparison_condition(self):
+        condition_data = self.create_event_frequency_condition(interval=self.interval)
+        condition_groups, group_id, unique_queries = self.create_condition_groups([condition_data])
+
+        results = get_condition_group_results(condition_groups, self.project)
+        assert results == {
+            unique_queries[0]: {group_id: 2},
+        }
+
+    def test_percent_comparison_condition(self):
+        condition_data = self.create_event_frequency_condition(
+            interval=self.interval,
+            comparison_type=ComparisonType.PERCENT,
+            comparison_interval=self.comparison_interval,
+        )
+        condition_groups, group_id, unique_queries = self.create_condition_groups([condition_data])
+        results = get_condition_group_results(condition_groups, self.project)
+
+        present_percent_query, offset_percent_query = unique_queries
+
+        assert results == {
+            present_percent_query: {group_id: 2},
+            offset_percent_query: {group_id: 1},
+        }
+
+    def test_count_and_percent_comparison_conditions(self):
+        count_data = self.create_event_frequency_condition(interval=self.interval)
+        percent_data = self.create_event_frequency_condition(
+            interval=self.interval,
+            comparison_type=ComparisonType.PERCENT,
+            comparison_interval=self.comparison_interval,
+        )
+        condition_groups, group_id, unique_queries = self.create_condition_groups(
+            [count_data, percent_data]
+        )
```

Comment:
thoughts on also adding an invalid condition to this list to ensure it processes as expected?